pygac/pod_reader.py

```diff
@@ -412,79 +417,106 @@ def decode_timestamps(encoded):
     def _get_times(self):
         return self.decode_timestamps(self.scans["time_code"])
 
-    def _adjust_clock_drift(self):
-        """Adjust the geolocation to compensate for the clock error.
+    def _compute_missing_lonlat(self, missed_utcs):
+        """compute lon lat values using pyorbital"""
+        tic = datetime.datetime.now()
 
-        TODO: bad things might happen when scanlines are skipped.
-        """
+        # TODO: Are we sure all satellites have this scan width in degrees ?
+        # TODO: Does this also work for LAC?
```

Comment:
> Are we sure all satellites have this scan width in degrees ?

According to Table J-1 (Satellite Scanning Instrument Parameters for TIROS-N through NOAA-14) in the KLM guide, the max scan angle for AVHRR/2 is given by 55.37 deg. I don't know where the `55.385` deg in the code come from, but the 55.37 deg is also the default in `pyorbita..geoloc_instrument_definitions.avhrr_gac`.

> Does this also work for LAC?

No, according to Table 8.3.1.3.1-1. (LAC/HRPT Data Characteristics.) the scan rate for LAC/HRPT is 360 scans per minute => 1/6 seconds/line instead of the 1/2 seconds/line which is currently assumed in `PODReader._adjust_clock_drift`. This can be fixed by making the scan rate a LAC/GAC reader attribute. This should be passed as frequency argument to `pyorbita..geoloc_instrument_definitions.avhrr_gac`.

Reply:
Just noticed that the GAC/LAC readers already have the attribute `reader.scan_freq`... I will definitely use it!

Reply:
> According to Table J-1 (Satellite Scanning Instrument Parameters for TIROS-N through NOAA-14) in the KLM guide, the max scan angle for AVHRR/2 is given by 55.37 deg. I don't know where the 55.385 deg in the code come from, but the 55.37 deg is also the default in pyorbita..geoloc_instrument_definitions.avhrr_gac.

I will have to check, but I think this was calculated somehow. It's somewhere in pyorbital maybe.